assertsprocessor/thresholds_test.go

```diff
@@ -137,14 +140,14 @@ func TestUpdateThresholds(t *testing.T) {
 		},
 		DefaultLatencyThreshold: 0.5,
 	}
-	var m = thresholdHelper{
+	var th = thresholdHelper{
 		logger:              logger,
 		config:              config,
 		thresholds:          &sync.Map{},
 		entityKeys:          &sync.Map{},
 		stop:                make(chan bool),
-		thresholdSyncTicker: clock.FromContext(ctx).NewTicker(time.Second),
-		assertsClient: &assertsClient{
+		thresholdSyncTicker: clock.FromContext(ctx).NewTicker(time.Millisecond),
```

Comment:
Do we really need millisecond level ticks in the test?

Reply:
I modified the ticks from seconds to millis in the unit tests. Now all the tests barely take a second to run. Earlier it took about 6 seconds